Thank you! Great work:+1::+1::+1:

Note: `gometalinter` returned a minor error, so I fixed a bit.

```console
go-arukas [ (old-and-new-port) ]$ make lint 

gofmt -s -l -w $(find . -name '*.go' | grep -v vendor)
gometalinter --vendor --skip=vendor/ --cyclo-over=15 --disable=gas --deadline=2m ./...
types.go:113:10:warning: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)
types.go:111::warning: declaration of "err" shadows declaration at ./types.go:108 (vetshadow)
make: *** [lint] Error 1
```

--- a/response_unmarshal_test.go
+++ b/response_unmarshal_test.go
@@ -11,7 +11,7 @@ func assertPort(t *testing.T, port *Port, number int32, protocol string) {
 	}
 }
 
-func TestUnmsharlOldPortFormat(t *testing.T) {
+func TestUnmarshalOldPortFormat(t *testing.T) {
 	responseBody := `
 		{
 			"data": {
@@ -39,7 +39,7 @@ func TestUnmsharlOldPortFormat(t *testing.T) {
 	assertPort(t, service.Ports()[2], 34197, "udp")
 }
 
-func TestUnmsharlNewPortFormat(t *testing.T) {
+func TestUnmarshalNewPortFormat(t *testing.T) {
 	responseBody := `
 		{
 			"data": {

